src/components/movie-view/MovieView.jsx

```diff
@@ -1,63 +1,80 @@
+  };
+
   if (!movie) return <div>Movie not found</div>;
 
   return (
     <div className="d-flex justify-content-center p-3"> 
-      <div style={{ maxWidth: '500px', width: '100%' }}>  
-        
-        <div className="mb-4 text-center">  
+      <div style={{ maxWidth: '800px', width: '100%' }} className="d-flex"> 
+        <div className="me-5"> 
           <img
             className="img-fluid"
             src={movie.ImagePath}
             alt={movie.Title}
-            style={{ maxWidth: '500px', height: 'auto' }}
+            style={{ maxWidth: '400px', height: 'auto' }}
           />
         </div>
 
-        <div className="mb-3">
-          <span className="fw-bold">Title: </span>
-          <span>{movie.Title}</span>
-        </div>
-        <div className="mb-3">
-          <span className="fw-bold">Description: </span>
-          <span>{movie.Description}</span>
-        </div>
-        <div className="mb-3">
-          <span className="fw-bold">Genre: </span>
-          <span>{movie.Genre.Name}</span>
-        </div>
-        <div className="mb-3">
-          <span className="fw-bold">Director: </span>
-          <span>{movie.Director.Name}</span>
-        </div>
-        <div className="mb-3">
-          <span className="fw-bold">Bio: </span>
-          <span>{movie.Director.Bio}</span>
-        </div>
-        <div className="mb-3">
-          <span className="fw-bold">Birth: </span>
-          <span>{movie.Director.Birth}</span>
-        </div>
-        <div className="mb-4">
-          <span className="fw-bold">Death: </span>
-          <span>{movie.Director.Death}</span>
-        </div>
+        <div className="flex-grow-1 d-flex flex-column">
+          {[
+            { label: 'Title', value: movie.Title },
+            { label: 'Description', value: movie.Description },
+            { label: 'Genre', value: movie.Genre.Name },
+            { label: 'Director', value: movie.Director.Name },
+            { label: 'Bio', value: movie.Director.Bio },
+            { label: 'Birth', value: movie.Director.Birth },
+            { label: 'Death', value: movie.Director.Death },
+          ].map((info, index) => (
+            <div className="mb-3" key={index}>
+              <span className="fw-bold">{info.label}: </span>
+              <span>{info.value}</span>
+              {info.label === 'Genre' && <hr className="my-2" />}
+            </div>
+          ))}
 
-        <div className="d-flex justify-content-center">
-          <Link to="/">
-            <Button variant="primary">Back</Button>
-          </Link>
+          <div className="mt-auto d-flex justify-content-between"> 
+            <Button
+              variant={isFavorite ? "secondary" : "primary"}
+              onClick={isFavorite ? null : addFavorite} // Disable if already a favorite
+              disabled={isFavorite}
+              className="w-100 me-2"
+            >
+              {isFavorite ? "Added to Favorites" : "Add to Favorites"}
```

Comment:
@Dbreazi ,

Looking at the logic you’ve laid out, regardless of whether a movie is marked as a favorite or not, the button name will always be “Added to Favorites.” It would be best to have this toggle between “Add to Favorites” and “Remove from Favorites.” If you include “Remove from Favorites,” it implies that you need to implement the function to perform that action.